outport/process/transactionsfee/utils.go

```diff
@@ -0,0 +1,67 @@
+package transactionsfee
+
+import (
+	"bytes"
+	"encoding/hex"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/smartContractResult"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"math/big"
+)
+
+const (
+	GasRefundForRelayerMessage = "gas refund for relayer"
+)
+
+func checkArg(arg ArgTransactionsFeeProcessor) error {
+	if check.IfNil(arg.TransactionsStorer) {
+		return ErrNilStorage
+	}
+	if check.IfNil(arg.ShardCoordinator) {
+		return ErrNilShardCoordinator
+	}
+	if check.IfNil(arg.TxFeeCalculator) {
+		return ErrNilTransactionFeeCalculator
+	}
+	if check.IfNil(arg.Marshalizer) {
+		return ErrNilMarshalizer
+	}
+
+	return nil
+}
+
+func isSCRForSenderWithRefund(scr *smartContractResult.SmartContractResult, txHash []byte, tx data.TransactionHandlerWithGasUsedAndFee) bool {
```

Comment:
to remove this util file, you can group all these functions in a component called: transactionChecker or something like this

Reply:
done